addons/mail/wizard/mail_compose_message.py

```diff
@@ -151,6 +152,46 @@ def _compute_reply_to_mode(self):
         for composer in self:
             composer.reply_to_mode = 'new' if composer.reply_to_force_new else 'update'
 
+    @api.depends('partner_ids')
+    def _compute_recipient_count(self):
```

Comment:
Question on task, as this is complicated and not correct. If we want to go that way, we have to re-use existing code making a match between followers and subtype. Moreover ``_message_get_default_recipients`` is used only with templates, and may create new partners based on jinja rendering. Aka: waaaay to much complicated according to me.